src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -103,6 +103,11 @@ @implementation GRPCCall {
 
 @synthesize state = _state;
 
+// TODO(jcanizales): If grpc_init is idempotent, this should be changed from load to initialize.
++ (void)load {
```

Comment:
Please ignore previous comment... I did not realize load() is a special function invoked automatically :)